libs/net/dal/Migrations/1.0.27/Up/PostUp/00-Notification.sql

```diff
@@ -0,0 +1,74 @@
+DO $$
+BEGIN
+
+UPDATE public."notification" SET
+  "settings" = regexp_replace(regexp_replace('{ "subject": "@inherits RazorEngineCore.RazorEngineTemplateBase<TNO.Services.Notification.Models.TemplateModel>
+@using System.Linq
+@{
+  var isAV = @Content.ContentType == TNO.Entities.ContentType.Snippet;
+  var isTranscriptAvailable = isAV && !string.IsNullOrWhiteSpace(@Content.Body) && @Content.IsApproved;
+  var sourceCode = !string.IsNullOrEmpty(@Content.Source?.Code) ? @Content.Source.Code : @Content.OtherSource;
+  var body = isTranscriptAvailable ? @Content.Body : (isAV ? @Content.Summary : @Content.Body);
+  var transcriptIcon = isTranscriptAvailable ? \"▶️📄\" : (isAV ? \"▶️\" : \"\");
+  var toneIcon = \"\";
+  switch (@Content.TonePools.FirstOrDefault()?.Value)
+  {
+    case 0:
+      toneIcon = \"😐 \";
+      break;
+    case -3:
+    case -4:
+    case -5:
+      toneIcon = \"☹️ \";
+      break;
+    case 3:
+    case 4:
+    case 5:
+      toneIcon = \"🙂 \";
+      break;
+  }
+}@toneIcon@sourceCode: @Content.Headline @transcriptIcon" }', '[\r\t]+', '', 'g'), '[\n]+', '\\n', 'g')::jsonb
+  , "template" = '@inherits RazorEngineCore.RazorEngineTemplateBase<TNO.Services.Notification.Models.TemplateModel>
+@{
+  var mmiaUrl = @MmiaUrl?.AbsoluteUri;
```

Comment:
Remove the NotificationOptions from the TemplateModel so that there was no dependency issues.